CityOfDetroit/bloom
# feat: add neighborhood amenities to public listing description

# Pull Request Template

## Issue Overview

This PR addresses #1459


## Description

Added neighborhood amenities section to listing details page (below map). Without any amenities provided it should not render that section. When some amenities provided it will only display those.

## How Can This Be Tested/Reviewed?

Go to listing, check if there is no neighborhood amenities. After that add those in partners page, and they should appear below map

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
